src/main/java/com/kodilla/ecommercee/domain/group/Group.java

```diff
@@ -0,0 +1,61 @@
+package com.kodilla.ecommercee.domain.group;
+
+import com.kodilla.ecommercee.domain.product.Product;
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "products_group")
+public class Group {
+
+    private Long groupId;
+    private Long groupName;
+    private List<Product> productsCategories = new ArrayList<>();
```

Comment:
`productsCategories` tłumacząc luźno na polski to "kategorie produktów", natomiast pole to przechowuje listę produktów. Proponuję nazwać to po prostu `products`

Reply:
Ale to uwaga do encji Group - poprawnie jest ona zrobiona - zgodnie z Twoimi uwagami w branchu JDP190802-13.

Reply:
OK, ale pamiętaj, ze bedziesz mial konflikty.